about.html

```diff
@@ -0,0 +1,23 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>About Page</title>
+    <link rel="stylesheet" href="../tech-shop/css/about.css">
+</head>
+<body>
+    <section id="about">
+    <h1>“We Aim To Be Our Stakeholders’ First Choice In Everything We Do”</h1>
+    <p><strong>Welcome to Tech-Shop</strong> <br>
+
+        We connect people and products – opening up a world of possibility. From bracelets and backpacks to tablets and toy cars – we give you access to everything you need and want. Our range is unparalleled, and our prices unbeatable.<br>
+        
+        Driven by smart technology, everything we do is designed to put the power directly in your hands – giving you the freedom to shop however, whenever and wherever you like.<br>
+        
+        We’re trusted by millions, because we don’t just deliver to your doorstep, we were born here. With Tech-Shop.com you’ll always be getting a good deal – with exceptional service that makes your shopping experience as easy and seamless as possible.<br>
+        
+        This is Tech-Shop – the power is in your hands.</p>
```

Comment:
Don't make the lines too long. The lines must at max from 80 to 100 characters
and don't put a white space as well the `p` tag will ignore it.